Dockerfile

```diff
@@ -1,34 +1,39 @@
 FROM        registry.access.redhat.com/ubi9/python-39:9.5-1739799514@sha256:9bbc4cfeac896544ab3eafa088c3d6995e82592362d374606f00d221f2986fe0 as prod
 
+COPY --from=ghcr.io/astral-sh/uv:0.6.11@sha256:fb91e82e8643382d5bce074ba0d167677d678faff4bd518dac670476d19b159c /uv /bin/uv
+
+COPY --chown=1001:0 pyproject.toml uv.lock README.md /dashdotdb/
 COPY        LICENSE /licenses/LICENSE
 USER        1001
 WORKDIR     /dashdotdb
-RUN         python3 -m venv venv
-ENV         VIRTUAL_ENV=/dashdotdb/venv
-ENV         PATH="$VIRTUAL_ENV/bin:$PATH"
+
+ENV \
+  UV_COMPILE_BYTECODE="true" \
+  UV_NO_CACHE=true
+RUN \
+  UV_DYNAMIC_VERSIONING_BYPASS="0.0.0" \
```

Comment:
What's this env variable doing?

Reply:
I think you can omit that. I believe it's tied to https://github.com/app-sre/qontract-reconcile/blob/master/pyproject.toml#L129-L139 which we aren't using for this project.